src/widgets/profile/relationships.card.test.tsx

```diff
@@ -39,6 +39,7 @@ describe("<RelationshipsCard/>", () => {
     expect(wrapper.getByText("\u2014")).not.toBeNull();
   });
 
+  /* TO BE UNCOMMENTED WHEN USING FHIR
```

Comment:
@reagan-meant  well done on this.Does this mean this section will always be uncommented when one wants to use FHIR,ie a non developer ?

Reply:
@HerbertYiga this is a test i comment hence it doesn't affect a non developer....Unfortunately you missed a background to this ticket....This is a temporally fix...[See here](https://talk.openmrs.org/t/fhir-url-for-relatedperson/29624)